package.json

```diff
@@ -11,8 +11,11 @@
     "lint": "eslint packages --ext .js,.ts,.tsx",
     "types:check": "tsc --skipLibCheck --noEmit",
     "test": "jest",
-    "test:ci": "jest --ci",
+    "test:ci": "jest --ci && yarn cypress:ci",
     "storybook": "start-storybook -p 9009 --ci",
+    "cypress:ci": "start-server-and-test storybook http://localhost:9009 cypress:run",
```

Comment:
Is the intention to run this only on CI then?

Reply:
To test locally we’d use `cypress:dev` for the interactive runner or `cypress:run` for headless, it’s implicit that storybook also needs to be running though. Happy to make it more explicit if we think it’d be easier.

Reply:
I’m ok with it as is t be honest